me.broken_by.PdfMetadataEditor.yml

```diff
@@ -0,0 +1,38 @@
+app-id: me.broken_by.PdfMetadataEditor
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+commands:
+  - "/app/bin/pmedit-cli"
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --filesystem=home
+  - --persist=.java
+modules:
+  - name: pdf-metadata-editor-app
+    buildsystem: simple
+    build-commands:
+      - mv -v * ${FLATPAK_DEST}
+    sources:
+      - type: archive
+        url: https://github.com/zaro/pdf-metadata-editor/releases/download/v3.0.0/pdf-metadata-editor-3.0.0-linux-portable.tar.gz
+        sha256: 80303e459c1c3b6ecf13923545ca349a2c3e1ff2a3ede08644adba98b2a9de2d
+  - name: pdf-metadata-editor-desktop
+    buildsystem: simple
+    build-commands:
+      - mkdir -p ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps
+      - mkdir -p ${FLATPAK_DEST}/share/mime/packages/
+      - mkdir -p ${FLATPAK_DEST}/share/applications/
+      - mkdir -p ${FLATPAK_DEST}/share/metainfo/
+      - cp -v flatpak/pdf-metadata-edit.png ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/me.broken_by.PdfMetadataEditor.png
+      - cp -v flatpak/me.broken_by.PdfMetadataEditor-mime.xml ${FLATPAK_DEST}/share/mime/packages/me.broken_by.PdfMetadataEditor.xml
+      - cp -v flatpak/me.broken_by.PdfMetadataEditor.desktop ${FLATPAK_DEST}/share/applications/
+      - cp -v me.broken_by.PdfMetadataEditor.metainfo.xml ${FLATPAK_DEST}/share/metainfo/
+    sources:
+      - type: archive
+        url: https://github.com/zaro/pdf-metadata-editor/archive/refs/heads/master.zip
```

Comment:
you are referencing `master` which mean if there is a commit between now and then, this will break.

Reply:
yes, I understand this is a problem. My intention was once all the other feedback is resolved to tag a new release and use it in place of master.

Reply:
its a chicken and egg problem because this needs to be resolved.

Reply:
As I said I will definitely resolve once it's the last item in the PR left.

I hope that is fine by you ?

Reply:
no. pin to a commit.

Reply:
Pinned to new release.